src/plugins/intel_cpu/src/utils/general_utils.h

```diff
@@ -205,4 +207,22 @@ inline bool all_of_values(const Container& container, const T& value) {
     });
 }
 
+template <typename To, typename From>
+inline std::enable_if_t<sizeof(To) == sizeof(From) && std::is_trivially_copyable_v<From> &&
```

Comment:
Let's move it under 'utils/cpp' folder and decide what implementation to use based on OPENVINO_CPP_VER macro